### DISCUSSION TITLE
Improvements to statistics card

### DISCUSSION THREAD

**Discussion by bmesuere** [home-assistant/frontend#10200] [Accessibility]
I really like the new statistics in home assistant, but I find the included statistics card a bit lacking.

- The order of the added entities is not respected for the order in the graph/legend
- The default colors are sometimes hard to interpret for people with color vision deficiencies
- The generated legend names can become very long (e.g. if only the mean is plotted, it is not needed to add "(mean)" to each of the labels

It would be great if the entities would accept an optional name and color in the configuration.

**Comment by bramkragten** [comment]
You can set the name of an entity:

```yaml
entities:
  - entity: entity.id
    name: Name
```
